activemodel/lib/active_model/dirty.rb

```diff
@@ -178,7 +178,7 @@ def changed_attributes
 
     # Handles <tt>*_changed?</tt> for +method_missing+.
     def attribute_changed?(attr, from: OPTION_NOT_GIVEN, to: OPTION_NOT_GIVEN) # :nodoc:
-      changes_include?(attr) &&
+      !!changes_include?(attr) &&
```

Comment:
What's going here?
Why casting to boolean is needed if `changes_include?` always returns true/false?

```
       # Returns +true+ if attr_name is changed, +false+ otherwise.
       def changes_include?(attr_name)
         attributes_changed_by_setter.include?(attr_name)
       end
```